### PR TITLE
Adding a file for HE, IL

### PR DESCRIPTION
**Pull request by melinmichael09**
This will add a file about melinmichael09's downtown.

### CODE REVIEW

File: places/melinmichael09-HE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## HE, IL
+
+### Places to eat
```
Comment (lcatlin) [inline places/melinmichael09-HE.md]: @melinmichael09 What are some of your favorite places to eat?

File: places/melinmichael09-HE.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+### Fun things to do
```
Comment (lcatlin) [inline places/melinmichael09-HE.md]: @melinmichael09 What are some of the fun things to do in your hometown?

File: places/melinmichael09-HE.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+### Awesome place to see
```
Comment (lcatlin) [inline places/melinmichael09-HE.md]: @melinmichael09 What are some of your favorite places to see

File: places/melinmichael09-HE.md (hunk 1 above)
Comment (melinmichael09) [inline places/melinmichael09-HE.md]: India House, Red Lobster, Yus Mandarin

**Pull request merged**